.github/workflows/ci.yml

```diff
@@ -0,0 +1,52 @@
+name: Run watir-rails CI
+
+'on':
+  pull_request:
+  push:
+    branches:
+      - master
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    continue-on-error: ${{ matrix.ruby-version == 'debug' }}
+    strategy:
+      matrix:
+        BUNDLE_GEMFILE: [gemfiles/Gemfile.rails-3.x, gemfiles/Gemfile.rails-4.x, Gemfile]
+        ruby-version: [2.6.3, debug]
```

Comment:
Can you explain what does `debug` version actually mean? Is it the current nightly?

Reply:
Yep, it's ruby-head with assertions enabled (see [here](https://github.com/ruby/setup-ruby#supported-versions)).